### ISSUE TITLE
CIs for `get_predicted.coxph()` when `type = "expected"`

### ISSUE THREAD

**Issue by strengejacke** [easystats/insight#531]
@vincentarelbundock @bwiernik for coxph, `type = "expected"` seems to return incorrect CIs - probably because the back-transformation is not correct. Do you know how to get CIs from the predictions and SEs, when `get_predicted(mod, predict = "expected")`?

From `?predict.coxph`:

> type the type of predicted value. Choices are (...) the expected number of events given the covariates and follow-up time ("expected") (...). The survival probability for a subject is equal to exp(-expected).

_Originally posted by @strengejacke in https://github.com/easystats/insight/issues/530#issuecomment-1066661597_

**Comment by vincentarelbundock** [comment]
It has been yeeeears since I've worked with these kinds of models, and I'm not sure exactly what should be done here. I can look into this issue eventually, but it's likely to take a couple weeks at least.

**Comment by strengejacke** [comment]
Ok, let's leave this open and work later on it.

**Comment by bwiernik** [comment]
Yeah, it's not just a inverse link for a survival model. Need to dig in to remember
